.travis.yml

```diff
@@ -76,12 +78,13 @@ matrix:
     - compiler: clang
       env: PRESET=ESP32 XI_MAKETARGET=""
     - compiler: gcc
-      env: TESTS=fuzztests PRESET=FUZZ_TESTS XI_BSP_TLS="" XI_MAKETARGET="fuzz_tests"
+      env: TESTS=fuzztests PRESET=FUZZ_TESTS XI_BSP_TLS="" XI_MAKETARGET="fuzz_tests" CXX=clang++
 before_script:
+  # https://wiki.ubuntu.com/SecurityTeam/Roadmap/KernelHardening#ptrace_Protection
   - "if [[ ! $TESTS == 'fuzztests' ]]; then
         if [[ $CC == *'clang'* ]]; then export CC='clang-3.8'; fi;
      else
-        if [[ $CC == *'clang'* ]]; then export CC=clang CXX=clang++; fi;
+        echo 0 | sudo tee /proc/sys/kernel/yama/ptrace_scope;
```

Comment:
If I build the tests on my personal computer, will they permanently remove the `ptrace` protection?
If that's the case, we should figure out how to set it back to the original value `after_script`?

Reply:
this affects only Travis temporary virtual linux machines. These machines will be destroyed afterwards.
Local macos dev machines aren't affected.
`.travis.yml` is used only by travis.

Reply:
d'oh, of course 😅  Thanks for clearing that up. PR Approved :+1: